lib/rubocop/cop/generator.rb

```diff
@@ -14,25 +14,45 @@ class Generator
         module RuboCop
           module Cop
             module %<department>s
-              # TODO: Write cop description and example of bad / good code.
+              # TODO: Write cop description and example of bad / good code. For every
+              # `SupportedStyle` and unique configuration, there needs to be examples.
+              # Examples must have valid Ruby syntax. Do not use upticks.
+              #
+              # @example EnforcedStyle: bar (default)
+              #   # Description of the `bar` style.
               #
-              # @example
               #   # bad
-              #   bad_method()
+              #   bad_bar_method()
```

Comment:
As not to be offensed by `Style/MethodCallWithoutArgsParentheses`, we should change it so that parentheses are not used when there is no argument.

```console
% cat /tmp/example.rb
# frozen_string_literal: true

foo()
% rubocop /tmp/example.rb
Inspecting 1 file
C

Offenses:

/tmp/example.rb:3:4: C: Style/MethodCallWithoutArgsParentheses: Do not use parentheses for method calls with no arguments.
foo()
   ^

1 file inspected, 1 offense detected
```